frankenstein/ring.go

```diff
@@ -111,3 +120,35 @@ func (r *Ring) updateSortedHashes() {
 	r.sortedHashes = hashes
 	r.circle = circle
 }
+
+// Describe implements prometheus.Collector.
```

Comment:
By which I mean, could you please a comment somewhere that explains why ring does this?